core/src/main/java/org/secondbase/core/SecondBase.java

```diff
@@ -1,13 +1,25 @@
 package org.secondbase.core;
 
 import org.secondbase.core.config.SecondBaseModule;
+import org.secondbase.flags.Flag;
 import org.secondbase.flags.Flags;
 
 /**
  * Coordination class for SecondBase modules. Handles Flags parsing and cooperation between modules.
  */
 public class SecondBase {
 
+    @Flag(
```

Comment:
Those are not flags-registered? And as such fail when running the example. Did you forget to check in changes in the SecondBase class?